eisbuk-admin/src/components/atoms/DateNavigation/DateNavigation.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { ComponentMeta } from "@storybook/react";
+import { DateTime } from "luxon";
+
+import DateNavigation from "./DateNavigation";
+import CopyPasteWeekButtons from "@/components/atoms/CopyPasteWeekButtons";
+
+export default {
+  title: "Date Navigation",
+  component: DateNavigation,
+} as ComponentMeta<typeof DateNavigation>;
+
+// we're using a standardized date not to fallback to current date and produce false positives with chromatic
+const defaultDateISO = "2021-03-01";
+const defaultDate = DateTime.fromISO(defaultDateISO);
+
+// ***** Region Different Timeframe ***** //
```

Comment:
What's your opinion on the following syntax? VSCode supports folding regions when it's used.

```javascript
// #region Foo
const some = "code that can be folded";
const together = "using a tick next to line 1"
// #endregion Foo
```

Reply:
Perfect! :ok_hand:

Reply:
I have fixed the syntax files touched here and will open up a ticket for other files